### ISSUE TITLE
user_db vs. users_dir

### ISSUE THREAD

**Issue by jvasile** [jvasile/Plinth#11]
The config file has settings for both user_db and users_dir.  What is the difference between the two?  This should either be documented or the settings should be merged.  Right now they are set to the same thing.

**Comment by tomgalloway** [comment]
The user changes I've been making are just using the db in user_db folder. I think the users_dir was an older implementation for text file list of users and can be removed.

**Comment by jvasile** [comment]
Yes, I think the TODO here is to get rid of the users_dir references in the code and then remove it from the config.
